mews-operations/reservations.md

```diff
@@ -248,6 +275,13 @@ The third `reservation` definition shows the partial cancellation - cancelling t
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `id` | `string` | optional | Identifier of company. |
+| `name` | `string` | optional | Name of company or travel agency. |
+| `contact` | [`Contact`](#contact) object | optional | Company or travel agency contact information. |
+
```

Comment:
| `name` | `string` | optional | Name of company. |
| `contact` | [`Contact`](#contact) object | optional | Company contact information. |